CHANGELOG.md

```diff
@@ -1,11 +1,48 @@
 # Changelog DeviceStatus
 ## Table of Contents
+- [r0.6](#r06)
 - [v0.5.1](#v051)
 - [v0.5.0](#v050)
 - [v0.5.0-rc](#v050-rc)
 - [v0.4.1](#v041)
 - [v0.2.0](#v020)
 
+# r0.6  
+## Please note:
+- This is an alpha version, it should be considered as a draft.
+- There are bug fixes to be expected and incompatible changes in upcoming versions.
+- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+
+## Device Status - v0.6.0-alpha.1
+- API definition **with inline documentation**: 
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceStatus/blob/release-0.6/code/API_definitions/device-status.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/release-0.6/code/API_definitions/device-status.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/release-0.6/code/API_definitions/device-status.yaml)
+### Main Changes
+* Endpoints renamed 
+* New response parameter lastStatusTime
+
+### Added
+* Addition of "lastStatusTime" Field by @sachinvodafone in https://github.com/camaraproject/DeviceStatus/pull/146
+* include x-correlator by @fernandopradocabrillo in https://github.com/camaraproject/DeviceStatus/pull/112
+
+### Changed
+* Change endpoint names to comply with guideline by @gmuratk in https://github.com/camaraproject/DeviceStatus/pull/131
+* renaming "EventType"-components to be more clear & update component descriptions by @maxl2287 in https://github.com/camaraproject/DeviceStatus/pull/141
+* Make '+' mandatory for phoneNumber by @bigludo7 in https://github.com/camaraproject/DeviceStatus/pull/144
+
+### Fixed
+* NA
+
+### Removed
+* NA
+  
+## New Contributors
+* NA
+
+**Full Changelog**: https://github.com/camaraproject/DeviceStatus/compare/v0.5.1...r0.6
+
+
 # v0.5.1
 **This is a bugfix release for the third alpha version of the CAMARA DeviceStatus API**
```

Comment:
I don't think that we want to fix the history, but according to the new nomenclature it would have been the "third initial version", not an alpha version.